tutorials/misc/plot_ecog.py

```diff
@@ -9,71 +11,114 @@
+# compensated below by the fact that we do not specify a trans file so the
+# Head<->MRI transform is the identity.
 
-mat = loadmat(mne.datasets.misc.data_path() + '/ecog/sample_ecog.mat')
-ch_names = mat['ch_names'].tolist()
-elec = mat['elec']  # electrode positions given in meters
-# Now we make a montage stating that the sEEG contacts are in head
-# coordinate system (although they are in MRI). This is compensated
-# by the fact that below we do not specicty a trans file so the Head<->MRI
-# transform is the identity.
-montage = mne.channels.make_dig_montage(ch_pos=dict(zip(ch_names, elec)),
-                                        coord_frame='head')
+montage = mne.channels.make_dig_montage(ch_pos, coord_frame='head')
 print('Created %s channel positions' % len(ch_names))
 
 ###############################################################################
-# Now that we have our electrode positions in MRI coordinates, we can create
-# our measurement info structure.
+# Now that we have our montage, we can load in our corresponding
+# time-series data and set the montage to the raw data.
+
+# first we'll load in the sample dataset
+raw = mne.io.read_raw_edf(misc_path + '/ecog/sample_ecog.edf')
+
+# drop bad channels
+raw.info['bads'].extend([ch for ch in raw.ch_names if ch not in ch_names])
+raw.load_data()
+raw.drop_channels(raw.info['bads'])
+
+# attach montage
+raw.set_montage(montage)
 
-info = mne.create_info(ch_names, 1000., 'ecog').set_montage(montage)
+###############################################################################
+# We then compute the signal power in certain frequency bands (e.g. 30-90 Hz).
+# We compute the power in gamma and alpha bands.
+gamma_power = np.sum(raw.copy().filter(30, 90).get_data() ** 2, axis=1)
+alpha_power = np.sum(raw.copy().filter(8, 12).get_data() ** 2, axis=1)
 
 ###############################################################################
 # We can then plot the locations of our electrodes on our subject's brain.
 #
 # .. note:: These are not real electrodes for this subject, so they
 #           do not align to the cortical surface perfectly.
 
-subjects_dir = mne.datasets.sample.data_path() + '/subjects'
-fig = plot_alignment(info, subject='sample', subjects_dir=subjects_dir,
+subjects_dir = sample_path + '/subjects'
+fig = plot_alignment(raw.info, subject='sample', subjects_dir=subjects_dir,
                      surfaces=['pial'])
 mne.viz.set_3d_view(fig, 200, 70)
 
 ###############################################################################
 # Sometimes it is useful to make a scatterplot for the current figure view.
 # This is best accomplished with matplotlib. We can capture an image of the
 # current mayavi view, along with the xy position of each electrode, with the
-# `snapshot_brain_montage` function.
+# `snapshot_brain_montage` function. We can then visualize for example the
+# gamma power on the brain using MNE and matplotlib functions.
```

Comment:
If you do move the snapshotting step earlier as I suggest, this paragraph will need to be tweaked/simplified. Something like "Now let's use matplotlib to overplot frequency band power onto the electrodes."